R/env.R

```diff
@@ -552,6 +595,26 @@ env_clone <- function(x, parent = env_parent(x)) {
   list2env(as.list(x, all.names = TRUE), parent = parent)
 }
 
+#' Does environment inherits from another environment?
```

Comment:
But I got this grammatical form from your early purrr documentation ;)

it feels a bit like contracted newspaper titles so it made sense to me.